tests/integration/static.test.ts

```diff
@@ -93,7 +93,7 @@ test<FixtureTestContext>('linked static resources are placed in correct place in
     ...Array.from(document('script[src]')).map((elem) => {
       return elem.attribs.src
     }),
-    ...Array.from(document('link[href]')).map((elem) => {
+    ...Array.from(document('link[href]:not([blocking="render"])')).map((elem) => {
```

Comment:
probably better to ignore `rel="expect"` instead since `blocking` may take other values.